### PR TITLE
 [1044] Added end-to-end tests at the end of Quickstart editorial

### PR DESCRIPTION
**Pull request by allincowell**
## Summary

> 🎯 **Purpose**: Added end-to-end tests in terms of assert statements for the issue: https://github.com/cleanlab/cleanlab/issues/1044

### CODE REVIEW

File: docs/source/tutorials/datalab/datalab_quickstart.ipynb [hunk 1]
```diff
@@ -746,6 +774,60 @@
...
+    "    s1 = set(l1)\n",
+    "    s2 = set(l2)\n",
+    "    intersect_set = s1.intersection(s2)\n",
+    "    union_set = s1.union(s2)\n",
+    "    if len(intersect_set) == 0:\n",
+    "        return 0\n",
+    "    return len(intersect_set) / len(union_set)\n",
+    "\n",
+    "identified_label_issues_indices = issue_results[issue_results[\"is_label_issue\"] == True].index.tolist()\n",
+    "label_issue_indices = issue_results[issue_results[\"given_label\"] != issue_results[\"predicted_label\"]].index.tolist()\n",
```
Comment (elisno) [inline docs/source/tutorials/datalab/datalab_quickstart.ipynb:812]: The `given_labels` are noisy, so testing the Jaccard similarity doesn't quite make sense here.

The `create_data()` function defined in the notebook should have "ground truth" labels that you can compare against.

File: docs/source/tutorials/datalab/datalab_quickstart.ipynb [hunk 2]
```diff
@@ -77,6 +77,7 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
+    "metadata": {},
```
[the remark this answers is not in the document]
Comment (allincowell) [reply docs/source/tutorials/datalab/datalab_quickstart.ipynb]: I have reverted these nested 'metadata' fields

File: docs/source/tutorials/datalab/datalab_quickstart.ipynb [hunk 3]
```diff
@@ -746,6 +774,60 @@
     "\n",
     "To learn more, check out this [example notebook](https://github.com/cleanlab/examples/blob/master/datalab_image_classification/datalab.ipynb) (demonstrates Datalab applied to a real dataset) and the [advanced Datalab tutorial](datalab_advanced.html) (demonstrates configuration and customization options to exert greater control)."
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "metadata": {}
+   },
```
[the remark this answers is not in the document]
Comment (allincowell) [reply docs/source/tutorials/datalab/datalab_quickstart.ipynb]: I have added "nbsphinx": "hidden" in metadata and shifted everything to 1 cell

File: docs/source/tutorials/datalab/datalab_quickstart.ipynb (hunk 1 above)
Comment (allincowell) [reply docs/source/tutorials/datalab/datalab_quickstart.ipynb]: I am now using y_train_idx as ground labels and noisy_labels_idx as predictions from create_data() function for comparision.

**Review by elisno — APPROVED** [review]
LGTM!

**Pull request merged**